docs/topic-guides/cluster-configuration/elasticsearch-logging-backend.txt

```diff
@@ -0,0 +1,56 @@
+.. _elasticsearch-logging-backend:
+
+##############################
+ Elasticsearch-backed logging
+##############################
+
+This guide covers the limitations of the default logging backend, as a
+guideline on when to migrate to Elastic, and some tips for how to tune
+Elasticsearch to work best with Determined.
+
+`Elasticsearch <https://www.elastic.co/what-is/elasticsearch>`__ is a
+search engine commonly used for storing application logs for search and
+analytics. Determined supports using Elasticsearch as the storage
+backend for trial logs. Configuring Determined to use Elasticsearch is
+simple, however managing an Elasticsearch cluster as scale is an
+involved task, so this guide is recommended for users who have hit the
+limitations of the default logging backend.
+
+Using the default logging backend, with a the standard deployment using
+``det-deploy``, the cluster can ingest logs about as fast as Postgres
+can persist them. For example, with ``det-deploy aws`` using Aurora
+Serverless with 2 capacity units, ingestion speed maxes out around 10-15
+MB/s (where the database's CPU hits ~90%). To get a little more mileage
+from the default, we recommend increasing the capacity of the database.
+At a certain point, the master instance itself will become the
+bottleneck, since it has limited incoming network bandwidth for HTTP
+requests delivering logs and limited resources to process them. For
+example, ``m5.large`` Amazon instances the max bandwidth is 0.74 Gb/s.
```

Comment:
Is this a limit defined/advertised by AWS or a value found through testing?

Reply:
It is a defined/advertised limit.